Description: Start in windowed mode by default
 Debian Games team has a preference for games starting in windoed mode, since
 fullscreen SDL will capture all keyboard input and make the user unable to
 escape via alt-tab, change volume, or exit sanely in the case of the
 application locking up.
Forwarded: not-needed
Author: Martin Erik Werner <martinerikwerner@gmail.com>
Last-Update: 2011-12-13

Index: redeclipse-1.3.1/src/engine/main.cpp
===================================================================
--- redeclipse-1.3.1.orig/src/engine/main.cpp	2012-08-31 15:43:28.000000000 +0200
+++ redeclipse-1.3.1/src/engine/main.cpp	2012-09-02 12:31:23.782739701 +0200
@@ -253,7 +253,7 @@
 #endif
 }
 
-VARF(0, fullscreen, 0, 1, 1, setfullscreen(fullscreen!=0));
+VARF(0, fullscreen, 0, 0, 1, setfullscreen(fullscreen!=0));
 
 void screenres(int *w, int *h)
 {
